content/docs/tech-insights/builtin-data-sources/index.md

```diff
@@ -10,21 +10,259 @@ description: Built-in Data Sources.
+* `DD_APP_TOKEN` [Read more](https://docs.datadoghq.com/api/latest/)
+
+<details>
+<summary><b> Facts </b></summary>
+
+| Name | Description          |
+| ------- | ------------------ |
+| Slo Count  | Number of SLOs configured in datadog matched based on the annotated tag.|
+| Monitor Count | Number of monitors configured in datadog with tags matching the monitor_tags annotation. |
+
+</details>
+
+### Dependabot Alerts Data Source
+
+Generates fact data from GitHub Dependabot Alerts for the entity.
+
+**Annotations used:** `github.com/project-slug`
+ 
+#### Authentication
+Authentication is done via [GitHub Apps](../../details/github-app-permissions/) or
+`GITHUB TOKEN` [Read more](../../integrations/github-token/)
+
+<details>
+<summary><b> Facts </b></summary>
+ 
+| Name | Description          |
+| ------- | ------------------ |
+| Open Alerts  | Number of Dependabot alerts with state "open".|
+| Dismissed Alerts | Number of Dependabot alerts with state "dismissed".|
+| Dismissed Without Reason Alerts | Number of Dependabot alerts with state "dismissed" that don\'t have dismissed_reason set.|
+| Dismissed Without Comment Alerts | Number of Dependabot alerts with state "dismissed" that don\'t have dismissed_comment set.|
+| Fixed Alerts |Number of Dependabot alerts with state "fixed".|
+| Open Critical Severity Alerts | Number of open Dependabot alerts with "critical" severity.|
+| Open High Severity Alerts | Number of open Dependabot alerts with "high" severity.|
+| Open Medium Severity Alerts | Number of open Dependabot alerts with "medium" severity.|
+| Open Low Severity Alerts | Number of open Dependabot alerts with "low" severity.|
+| Dissmissed Critical Severity Alerts | Number of dismissed Dependabot alerts with "critical" severity.|
+| Dissmissed High Severity Alerts | Number of dismissed Dependabot alerts with "high" severity.|
+| Dissmissed Medium Severity Alerts | Number of dismissed Dependabot alerts with "medium" severity.|
+| Dissmissed Low Severity Alerts | Number of dismissed Dependabot alerts with "low" severity.|
+| Fixed Critical Severity Alerts | Number of fixed Dependabot alerts with "critical" severity.|
+| Fixed High Severity Alerts | Number of fixed Dependabot alerts with "high" severity.|
+| Fixed Medium Severity Alerts | Number of fixed Dependabot alerts with "medium" severity.|
+| Fixed Low Severity Alerts | Number of fixed Dependabot alerts with "low" severity.|
+| Oldest Open Alert Publish Date| Oldest alert publish date with state "open".|
+| Oldest Open Alert Update Date| Oldest alert update date with state "open".|
+
+</details>
+
+### Entity Metadata Data Source
+
+Generates facts for entities you have loaded into Backstage based on the data defined in their entity YAML.
+
+**Annotations used:** None, applicable to entities which are component, domain, system, api, resource or template kind.
+
+#### Authentication
+Authentication is done via [GitHub Apps](../../details/github-app-permissions/) or
```

Comment:
This one needs a bit more explanation. I actually don't understand myself. When does it use the GitHub app and when does it use the token? Does it use both?